device/isis/isis.yaml

```diff
@@ -57,3 +57,10 @@ components:
         name:
           x-include: ../common/common.yaml#/components/schemas/Named.Object/properties/name
           x-field-uid: 9
+        segment_routing:
+          description: >-
+            Segment Routing (SR).
+          $ref: './segmentrouting.yaml#/components/schemas/Isis.SegmentRouting'
```

Comment:
Can we put this class under IsisInterface instead of Isis since this is interface specific ?

Reply:
This is used under isis itself so is fine as is.